### DISCUSSION TITLE
Youtube not working on a few videos

### DISCUSSION THREAD

**Discussion by Arcanus6669** [aclap-dev/video-downloadhelper#1189] [Help and Q&A]
### Your question

Most of the time it works perfectly but so far I have come across 2 videos that do not trigger any video available for download.
https://www.youtube.com/watch?v=Mw8GbmFe14E
https://www.youtube.com/watch?v=QdO_IhDaFSo

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"d82dc315f25c473d"}
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "valid_license_message_has_been_displayed": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "yt_warning": false,
    "open_count_store": 4,
    "successfull_dl": 21,
    "never_show_successfull_dl_message": true,
    "download_history": {
        "__serializer_tag": "map",
        "__serializer_value": []
    }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

You will find a number of previous discussions in here to show that you are reporting a duplicate problem.  It is well known that VDH does not work on age restricted YouTube content.  It also doesn't work when someone is logged in on YouTube with a paid account.  We are all eagerly awaiting a solution from @paulrouget Paul.
